app/styles/components/_grid.scss

```diff
@@ -1,11 +1,46 @@
-$grid-max-width: 1200px;
+$grid-max-width: 950px; // (person-image + person-information * 2) + grid-cell Right Margin
+$grid-break-mobile: 1049px; 
 
 .Grid {
   display: flex;
   flex-direction: row;
   flex-wrap: wrap;
-  justify-content: space-between;
+  justify-content: center;
 
   max-width: $grid-max-width;
-  margin-top: $Theme-margin-half;
+}
+
+.Grid-cell {
+  margin-bottom: $Theme-margin-default;
+}
+
+.Grid-fullWidthCell {
+  flex: 1;
+}
+
+.Grid-cell:nth-child(odd) {
+  margin-right: $Theme-margin-default;
+}
+
+.Grid-cell:nth-last-child(2),
+.Grid-cell:nth-last-child(1) {
+  margin-bottom: 0;
+}
```

Comment:
I confirmed with @fcbaila that this Grid is not supposed to have more than two columns, with any new members to be added into a new row. 
I think this validates these assumptions, which allows to remove some repetition on the html, specially if more board members of officers are added in the future.